### PR TITLE
[bitnami/redmine] Add how to deploy in a different URI to the README.md

### PR DESCRIPTION
**Pull request by dfgrosso**
Since extraVolumes and extraVolumeMounts were added to the chart template it is now easier do deploy the chart under a sub-uri. This might be helpful to the community.

**Description of the change**

**Benefits**

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [x] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

**Comment by dfgrosso** [comment]
Hello. Its my first contribution here. Can someone help me fix this  "lint" error.

Thank you

**Comment by dani8art** [comment]
Hi @dfgrosso, you would need to bump the version of the chart to fix the linter issue.

**Comment by dfgrosso** [comment]
thanks dani8art..what a noobe :)

File: bitnami/redmine/README.md [hunk 1]
```diff
@@ -36,6 +36,91 @@ The command deploys Redmine on the Kubernetes cluster in the default configurati
 
 > **Tip**: List all releases using `helm list`
 
+## Deploying to a sub-URI
```
Comment (dani8art) [inline bitnami/redmine/README.md:39]: Maybe this kind of context could fit better in [this section](https://github.com/bitnami/charts/tree/master/bitnami/redmine#configuration-and-installation-details), don't you think? Could you put it at the end of that section?

**Review by dani8art — CHANGES_REQUESTED** [review]

File: bitnami/redmine/README.md (hunk 1 above)
Comment (dfgrosso) [reply bitnami/redmine/README.md:39]: Hello.

Agree ;)

I'll do that!

File: bitnami/redmine/README.md [hunk 2]
```diff
@@ -328,6 +328,91 @@ Bitnami will release a new chart updating its containers if a new version of the
 
 Redmine writes uploaded files to a persistent volume. By default that volume cannot be shared between pods (RWO). In such a configuration the `replicas` option must be set to `1`. If the persistent volume supports more than one writer (RWX), ie NFS, `replicas` can be greater than `1`.
 
+### Deploying to a sub-URI
+
+(adapted from https://hub.docker.com/r/bitnami/redmine/)
```
Comment (dani8art) [inline bitnami/redmine/README.md:333]: please use the github link instead, [https://github.com/bitnami/bitnami-docker-redmine](https://github.com/bitnami/bitnami-docker-redmine)

**Review by dani8art — COMMENTED** [review]
LGTM, just a minor comment.

File: bitnami/redmine/README.md (hunk 2 above)
Comment (dfgrosso) [reply bitnami/redmine/README.md]: done

**Review by dani8art — APPROVED** [review]
LGTM, Thanks!

**Pull request merged**